packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -330,6 +332,23 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
     }
   }
 
+  private _setTargetWidthAsync() {
```

Comment:
@joschect Here's the attempt at asynchronously getting the target's width. Let me know what you think.